### PR TITLE
added sql commands

### PR DESCRIPTION
**Pull request by jazzaik**
Created tables and added to the magflora_archive_manager server

### CODE REVIEW

File: tables/create_archive_manager.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE archive_manager
+    (   first_name VARCHAR(32),
+        last_name VARCHAR(32), 
+        handle VARCHAR(16), 
+        email  VARCHAR(255), 
+        access_level VARCHAR(32),
```
Comment (indirectjump) [inline tables/create_archive_manager.sql]: We need another datatype here. This column needs to only select one of the access levels predefined. Think of this as similar to drop-down list. Find the appropriate datatype from MySQL documentation.

File: tables/create_archive_manager.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE archive_manager
+    (   first_name VARCHAR(32),
+        last_name VARCHAR(32), 
+        handle VARCHAR(16), 
+        email  VARCHAR(255),
```
Comment (indirectjump) [inline tables/create_archive_manager.sql]: We also need add email validation. Please do research on how to add that.

File: tables/create_archive_manager.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE archive_manager
+    (   first_name VARCHAR(32),
+        last_name VARCHAR(32), 
+        handle VARCHAR(16), 
+        email  VARCHAR(255), 
+        access_level VARCHAR(32), 
+        date_joined  DATE,
```
Comment (indirectjump) [inline tables/create_archive_manager.sql]: See if we can do this,
We need a default value here, which is the date when we insert the values in this table.

File: tables/create_archive_manager.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+        active VARCHAR(32),
```
Comment (indirectjump) [inline tables/create_archive_manager.sql]: Rename this field to `is_active`. Change datatype to binary/boolean type.

File: tables/create_database_manager.sql [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+(   first_name VARCHAR(32),
+    last_name VARCHAR(32),
+    handle VARCHAR(16), 
+    email  VARCHAR(255), 
+    access_level VARCHAR(32),
+    date_joined  DATE,
+    active VARCHAR(32), 
+    date_left DATE);
+
+    ALTER TABLE database_manager
```
Comment (indirectjump) [inline tables/create_database_manager.sql]: Merge primary key clause in CREATE.

File: tables/create_database_manager.sql [hunk 6]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE database_manager
+(   first_name VARCHAR(32),
+    last_name VARCHAR(32),
+    handle VARCHAR(16), 
+    email  VARCHAR(255), 
+    access_level VARCHAR(32),
+    date_joined  DATE,
+    active VARCHAR(32),
```
Comment (indirectjump) [inline tables/create_database_manager.sql]: Same comment as first table.

File: tables/create_database_manager.sql [hunk 7]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE database_manager
+(   first_name VARCHAR(32),
+    last_name VARCHAR(32),
+    handle VARCHAR(16), 
+    email  VARCHAR(255), 
+    access_level VARCHAR(32),
```
Comment (indirectjump) [inline tables/create_database_manager.sql]: Same comment as first table.

File: tables/create_specimen.sql [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+     id INT,
+     specimen_type VARCHAR(50),
+     plant_id VARCHAR(50),
+     location VARCHAR(512),
+     preservation_type VARCHAR(225),
+     original_dimension VARCHAR(225),
+     color_data VARCHAR(50),
+     collected_from VARCHAR(50),
+     collected_by VARCHAR(50),
+     collected_on VARCHAR(50),
```
Comment (indirectjump) [inline tables/create_specimen.sql]: Incorrect datatype. This needs to be DATE. Similarly for other fields which ends with `on`.

File: tables/create_specimen.sql [hunk 9]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE specimen(
+     id INT,
+     specimen_type VARCHAR(50),
+     plant_id VARCHAR(50),
+     location VARCHAR(512),
+     preservation_type VARCHAR(225),
```
Comment (indirectjump) [inline tables/create_specimen.sql]: Needs to be a different datatype. Similar to Access levels in the first table.

File: tables/create_specimen.sql [hunk 10]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE specimen(
+     id INT,
+     specimen_type VARCHAR(50),
+     plant_id VARCHAR(50),
+     location VARCHAR(512),
```
Comment (indirectjump) [inline tables/create_specimen.sql]: Let's break location into two different fields, Geo-coordinates and location name.

File: tables/create_specimen.sql [hunk 11]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE specimen(
+     id INT,
+     specimen_type VARCHAR(50),
```
Comment (indirectjump) [inline tables/create_specimen.sql:3]: Need different datatype here. Similar to Access levels in the first table.

**Review by indirectjump — CHANGES_REQUESTED** [review]
Follow changes as per the comments. Most comments apply to more than one file.

File: tables/create_archive_manager.sql (hunk 1 above)
Comment (jazzaik) [reply tables/create_archive_manager.sql]: I added a constraint making sure that it only accepts AR-L1,AR-L2, AR-L3, etc.,
ALTER TABLE archive_manager
    -> ADD CONSTRAINT chk_lvl CHECK(access_level in ('AR-L1', 'AR-L2', 'AR-L3','AR-L4'));

**Review by indirectjump — COMMENTED** [review]
Accepting this PR with partial review to speed into v1

**Review by indirectjump — APPROVED** [review]

**Pull request merged**